### PR TITLE
helm charts: increase failure threshold for liveness and readiness probes

### PR DESCRIPTION
**Pull request by dmitsh**
# Description

helm charts: increase failure threshold for liveness and readiness probes in dapr placement deployment

## Issue reference

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: charts/dapr/charts/dapr_placement/templates/dapr_placement_deployment.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ spec:
           initialDelaySeconds: {{ .Values.debug.initialDelaySeconds }}
 {{- end }}
           periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
-          failureThreshold: 5
+          failureThreshold: 30
```
Comment (daixiang0) [inline charts/dapr/charts/dapr_placement/templates/dapr_placement_deployment.yaml:49]: What is the root cause of failure? If related to workloads, we should add it as a configurable value.

**Review by artursouza — APPROVED** [review]

**Pull request merged**